src/main/java/com/novi/backend_spring_boot_tech_it_easy_controller/controllers/TelevisionsController.java

```diff
@@ -0,0 +1,38 @@
+package com.novi.backend_spring_boot_tech_it_easy_controller.controllers;
+
+import com.novi.backend_spring_boot_tech_it_easy_controller.models.TelevisionRequest;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+
+@RequestMapping("/api/televisions")
+public class TelevisionsController {
+
+    @GetMapping
+    @ResponseBody
+    public String getAllTelevisions() {
+        return "List Televisions";
+    }
+
+    @GetMapping("/{model}")
+    @ResponseBody
+    public ResponseEntity<String> getTelevision(@PathVariable String model) {
```

Comment:
Ik verwacht hier als return waarde "ResponseEntity<Television>" en als pathvariable dus "Long id" , zoals in je entity model